src/_includes/navbar.njk

```diff
@@ -30,7 +30,8 @@
             "name": "Program",
             "hasSubmenu": true,
             "submenu": [
-              {"name": "Lightning Talks", "link": "program/lightning-talks"}
+              {"name": "Schedule", "link": "program/schedule"},
+              {"name": "Lightning Talks", "link": "program/lightning-talks"} 
```

Comment:
The Schedule should stay in the top navigation as well as the submenu since Schedule is an important CTA.

The page link should be `program/schedule` for both the Schedule nav items.